Branch-SDK/Branch-SDK/BNCCrashlyticsWrapper.m

```diff
@@ -0,0 +1,71 @@
+//
+//  BNCCrashlyticsReportingHelper.m
+//  Branch.framework
+//
+//  Created by Jimmy Dee on 7/18/17.
+//  Copyright © 2017 Branch Metrics. All rights reserved.
+//
+
+#import "BNCCrashlyticsWrapper.h"
+
+@interface BNCCrashlyticsWrapper()
+@property (nonatomic, nullable) id crashlytics;
+@end
+
+@implementation BNCCrashlyticsWrapper
+
++ (id)sharedInstance
```

Comment:
Your style is legible and legit.

Just so you know, we've been using Google style bracing in the code base rather than Apple style.  I don't care as long as it's consistent through through a section of code.

I have a clang-format script that I've made but haven't used that enforces a consistent coding style on the code base.  We can talk about the enforcing a style but I find _*legible*_ is most important, and having a strict style just creates unnecessary friction.